src/main/java/io/fabric8/maven/docker/util/VolumeBindingUtil.java

```diff
@@ -181,6 +195,13 @@ public static String resolveRelativeVolumeBinding(File baseDir, String bindingSt
             } catch (IOException e) {
                 throw new RuntimeException("Unable to canonicalize '" + resolvedFile + "'");
             }
+        } else if (SystemUtils.IS_OS_WINDOWS && (localPath.contains(DOUBLE_DOT))) {
```

Comment:
I think you can always resolve the absolut path, regardless wether its on Window/Linux or whether it contains .. or not. In the worst case its a no-operation, but it would be easier to read.

Reply:
My first try was exactly this, but doing so make cause some test failures and I don't know this project enough to understand how to fix the additional failures